lib/Backend/InliningDecider.cpp

```diff
@@ -624,26 +624,8 @@ bool InliningDecider::GetBuiltInInfoCommon(
         break;
 #endif
 
-#ifdef ENABLE_SIMDJS
-    // SIMD_JS
-    // we only inline, and hence type-spec on IA
-#if defined(_M_X64) || defined(_M_IX86)
     default:
-    {
-#if 0 // TODO OOP JIT, inline SIMD
-        // inline only if simdjs and simd128 type-spec is enabled.
```

Comment:
Since we haven't removed any SIMD specific code so far I think we should keep the code and fix this by using #else when ENABLE_SIMDJS is off